index.html

```diff
@@ -1177,6 +1179,13 @@
             sync (unless <a>media mirroring</a> is used).
           </div>
           <div class="note">
+            The user agent is encouraged to pass locale information to the
+            <a>remote playback device</a> when possible, so that the <a>remote
+            playback device</a> may adjust its user interface and operational
+            functions to locale-specific attributes that reflect the user's
+            preferences.
```

Comment:
The Presentation API specifically asks the presentation receiver to set HTTP headers consistent with the controlling user agent's locale.

https://w3c.github.io/presentation-api/#creating-a-receiving-browsing-context

For remote playback, these headers could impact which media resource is fetched by the remote playback device (in the flinging case), if different media resources have locale-specific embedded text cues.

Reply:
See previous comment, I included this as an example of locale usage by the remote playback device.